assembl/static2/js/app/components/debate/common/announcement.jsx

```diff
@@ -18,8 +18,9 @@ export const createTooltip = (sentiment: SentimentDefinition, count: number) =>
 
 type SentimentsCounts = {
   [string]: {
-    ...SentimentDefinition,
-    count: number
+    count: number,
+    color: string,
+    Tooltip: React.Node
```

Comment:
This change is wrong. You need the full `...SentimentDefinition` here, cameType is used in `createTooltip` via `createDoughnutElements`

Reply:
even though SentimentDefinition declaration in sentimentDefinitions.jsx declares ```color: string``` down in createDoughtnutElements there is an error on ```color: sentimentCounts[key].color``` if I don't explicitely declare it in type SentimentCounts :( 
I think flow has trouble with mapping over Object.keys